src/lua/utils.h

```diff
@@ -218,6 +218,13 @@ luaL_checkcdata(struct lua_State *L, int idx, uint32_t *ctypeid);
 LUA_API void
 luaL_setcdatagc(struct lua_State *L, int idx);
 
+/**
+ * @brief Return size of currently allocated memory.
+ * @param L Lua State
+ */
+LUA_API int
```

Comment:
There is not a public function, so please don't use LUA_API here.

Reply:
Corrected